### ISSUE TITLE
Ruby library release — gem is currently a security placeholder

### ISSUE THREAD

**Issue by reidab** [heremaps/flexible-polyline#88]
The Ruby version of this library makes reference to installing with `gem install flexpolyline` and is set up as a gem with that name. 

However, the published version of this package at https://rubygems.org/gems/flexpolyline/ is an empty placeholder set up by @mensfeld of the RubyGems Security Team, presumably because this library was never actually published.

I believe someone from this project needs to contact RubyGems to regain control of the package and then cut a working release so that the library is accessible.

**Comment by mensfeld** [comment]
@reidab, yes, exactly. I did this because packages like this are frequent target to brandjacking. Happy to hand it to the team :)
